bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -1939,4 +1941,27 @@ public ServerConfiguration setNettyMaxFrameSizeBytes(int maxSize) {
         super.setNettyMaxFrameSizeBytes(maxSize);
         return this;
     }
+
+    /**
+     * returns whether it is allowed to have multiple ledger/index Directories in the same 
+     * filesystem partition
+     *
+     * @return  
+     */
+    public boolean isAllowMultipleDirsUnderSamePartition() {
```

Comment:
I feel ‘partition’ is more appropriate than ‘disk’. Term ‘disk’ is overloaded and theoretically there can be multiple partitions in a disk. Even java’s FileStore API https://docs.oracle.com/javase/7/docs/api/java/nio/file/FileStore.html says FileStore represents ‘partition’.

In any case, if you feel strong about it, I can use term ‘DiskPartition’.